src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -43,6 +41,7 @@ export const MessagingAttachment: React.FC<Props> = ({
         <button
           className={cssClass("CloseButton")}
           onClick={(e) => handleRemoveClick(e, onRemoveAttachment, attachmentID)}
+          aria-label={`close attachment with title: "${title}" and ID: ${attachmentID}`}
```

Comment:
Awesome to see, thank you! Might remove the `and ID: ${attachmentID}` portion though? We don't expose that in the visuals of the UI, so no need here - don't think having a whole UUID read by the screen-reader will actually help, just the title seems like plenty.